js/rendererjs/loadingScreen.js

```diff
@@ -106,6 +106,18 @@ export default async function loadingScreen(initUI) {
 		showError(e.toString())
 		return
 	}
+
+	// Set a timeout to display a warning message about long load times caused by rescan.
+	setTimeout(() => {
+		if (overlayText.textContent === 'Loading Sia...') {
+			overlayText.innerHTML= 'Loading can take a while after upgrading to a new version. Check the <a style="text-decoration: underline; cursor: pointer" id="releasenotelink">release notes</a> for more details.'
+
+			document.getElementById('releasenotelink').onclick = () => {
+				shell.openExternal('https://github.com/NebulousLabs/Sia/releases/tag/v1.1.1')
```

Comment:
we can probably just use `/releases`. Would be a pain to update this link on each release; we might forget.